### DISCUSSION TITLE
Documentation for BlockUI

### DISCUSSION THREAD

**Discussion by aboudard** [primefaces/community#2603] [PrimeNG] [answered]
In order to be as clear as possible, the documentation page of BlockUI should mention that the component referenced by the `target` binding should implement `BlockableUI`. Maybe a list of components could be extracted from the source code.

That would help developers avoid runtime errors when trying to target any element / component.

https://primeng.org/blockui
It could be mentioned in every version that has this BlockUI feature.

**Comment by qwadrox** [comment, marked as answer]
You're right. I created an issue. https://github.com/primefaces/primeng/issues/16317
